tokenserver/assignment/__init__.py

```diff
@@ -16,6 +21,8 @@ def get_user(self, service, email):
           * generation:  the last-seen generation number for that email
           * client_state:  the last-seen client state string for that email
           * old_client_states:  any previously--seen client state strings
+          * migration_state: one of None, "MIGRATING", "MIGRATED", where
+            a state of "MIGRATING" should cause a 501 return.
```

Comment:
The way you use this field does not seem to match the way its described here.

Reply:
Yeah, I was thinking further ahead, and missed the state for a new user that is being sent to spanner as part of the migration test. I'll fix this to include the transition.